tensorflow/c/tensor_shape_utils_test.cc

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Looks like there are two test files:
`tensor_shape_utils_test.cc` and `tf_shape_utils_test.cc`. I think you planned to remove one of them.